polkadot/node/core/runtime-api/src/cache.rs

```diff
@@ -444,6 +446,21 @@ impl RequestResultCache {
 		self.minimum_backing_votes.insert(session_index, minimum_backing_votes);
 	}
 
+	pub(crate) fn disabled_validators(
+		&mut self,
+		relay_parent: &Hash,
+	) -> Option<&Vec<ValidatorIndex>> {
+		self.disabled_validators.get(relay_parent).map(|v| &*v)
```

Comment:
Out of curiosity, what that `.map(|v| &*v)` does?

Reply:
It converts &mut to &